packages/connections/src/stores/connections-store.ts

```diff
@@ -222,6 +270,7 @@ export function useConnections(
           connectionAttempt: newConnectionAttempt,
         });
 
+        trackConnectionAttemptEvent(connectionInfo);
```

Comment:
what would happen if `trackConnectionAttemptEvent` throws?

Reply:
don't expect this to throw.